src/sage.ts

```diff
@@ -1,7 +1,7 @@
 import 'module-alias/register';
 import consoleStamp from 'console-stamp';
 import { MongoClient } from 'mongodb';
-import { ApplicationCommandPermissionData, Client, ExcludeEnum, Intents, PartialTypes, Team } from 'discord.js';
+import { ApplicationCommandPermissionData, Client, ExcludeEnum, Intents, PartialTypes } from 'discord.js';
 import { readdirRecursive } from '@root/src/lib/utils/generalUtils';
```

Comment:
I know you didn't do this originally but can you pls:
```suggestion
import { readdirRecursive } from '@lib/utils/generalUtils';
```